Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.py

```diff
@@ -270,7 +270,7 @@ def create_new_folder(self, object_type, object_type_id, parent_id, folder_name)
         Create a new folder in a Drive with a specified parent item or path.
         :param object_type: ms graph resource.
         :param object_type_id: the selected object type id.
-        :param parent_id: an ID of the Drive to upload the folder to.
+        :param parent_id: an ID of the ￿parent to upload the folder to.
```

Comment:
```suggestion
        :param parent_id: an ID of the parent to upload the folder to.
```
should solve the yml failure